src/main/java/seedu/duke/command/ListNoteCommand.java

```diff
@@ -111,21 +113,22 @@ public String execute() {
         StringBuilder unpinnedNotesSorted;
         ArrayList<Note> notes = new ArrayList<>();
         ArrayList<Note> pinnedNotes = new ArrayList<>();
-        ArrayList<Note> archivedNotes;
         ArrayList<Note> unpinnedNotes = new ArrayList<>();
+        ArrayList<Note> archivedNotes = new ArrayList<>();
```

Comment:
Is there a need to new the arrayList here? Ideally, ListNoteCommand should just call a function in notebook, say getNotes(bool isSorted, arrayList<Tag>) (currently is only getNotes(), you can overload it). All logic handling should be within notebook class itself to sort and return the requested arraylist of note. This is also for reusability, having the function in the notebook instead in this ListNoteCommand Class. Do refer to the word doc I shared.